src/controllers/user.controller.js

```diff
@@ -0,0 +1,109 @@
+import { ApiError } from '../exeptions/api.error.js';
+import { User } from '../modules/User.js';
+import { userService } from '../service/user.service.js';
+import bcrypt from 'bcrypt';
+
+const getAllActivated = async (req, res) => {
+  const users = await userService.getAllActivated();
+
+  res.send(users.map(userService.normalize));
+};
+
+const changeName = async (req, res) => {
+  const { userName } = req.params;
+  const { newName } = req.body;
+
+  if (!userName || !newName) {
+    throw ApiError.badRequest('Something wrong');
```

Comment:
The error message 'Something wrong' is too vague. Consider providing a more descriptive error message to help users understand what went wrong.